src/zenml/cli/model.py

```diff
@@ -375,7 +368,7 @@ def list_model_versions(model_name_or_id: str, **kwargs: Any) -> None:
         **kwargs: Keyword arguments to filter models.
     """
     model_id = Client().get_model(model_name_or_id=model_name_or_id).id
-    model_versions = Client().list_model_versions(
+    model_versions = Client().zen_store.list_model_versions(
```

Comment:
Because I need the response model back, not `ModelVersion`.

Reply:
As suggested in the other comment - the Client should return the response model anyway if we want to be consistent with the current Client interface, so I guess this would also solve this issue then?

Reply:
Prefer to keep it as is, no one even knows about zenml.models.model_models.ModelVersionResponseModel thing - it is too internal to be a concerning, in my opinion.